### ISSUE TITLE
`prefer-minimal-ternary`: suggestion to exclude non-computed property access like import namespaces

### ISSUE THREAD

**Issue by lukpsaxo** [sindresorhus/eslint-plugin-unicorn#3297]
### Description

prefer-minimal-ternery rule description sounds good to me, but when I run it on our codebase I get alot of hits like this:

```
    const defaultSelectedTabId =
        !showTabs && x
            ? constants.TAB_A
            : constants.TAB_B;

```

I guess it is suggesting it should be

```
    const defaultSelectedTabId =
        constants[
            !showTabs && x
            ? 'TAB_A'
            : 'TAB_B'
        ];
```

and that isn't one of the examples given in the rule description - I personally would exclude non-computed object property lookup from the rule, especially since it is enabled in the un-opiniated config and for me the suggested version looks worse than the original.

### Examples

```js
// ❌
var replace = !x ? c[x] : c[y];

// ✅
var replace = c[!x ? x : y];

// ✅
var replace = !x ? c.x : c.y;
```

**Comment by sindresorhus** [comment]
This was already fixed in https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v67.0.0
